config/build/rbac/aggregated_roles.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: role
+  labels:
+    # Add these permissions to the "admin" and "edit" default roles.
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+rules:
+- apiGroups:
+  - build.knative.dev
```

Comment:
do we want to do this for the apiGroup `build.projectriff.io` as well?

Reply:
that's an epic typo